From 197f7ba0e5fe6136c391beddb30d69888931785b Mon Sep 17 00:00:00 2001
From: Alan Pevec <apevec@redhat.com>
Date: Sat, 30 Jun 2012 01:00:25 +0200
Subject: [PATCH] match egg and spec requires

Upstream build process generates egg requires from tools/pip-requires
which are nailed for the upstream CI needs.
This patch adjusts deps so that generated egg-info requires can be
satisfied by the system packaged Python libraries.

FEDORA SPECIFIC - do not send upstream!

Change-Id: I8c6a1c73c1b1c6edbf507cb5f0ba91f3f97cdd72
---
 tools/pip-requires |    8 ++++++--
 1 files changed, 6 insertions(+), 2 deletions(-)

diff --git a/tools/pip-requires b/tools/pip-requires
index ec5562d..22f4784 100644
--- a/tools/pip-requires
+++ b/tools/pip-requires
@@ -1,6 +1,10 @@
 # keystone dependencies
-pam==0.1.4
-WebOb==1.0.8
+# adjust deps so that generated egg-info matches Requires: in spec
+PyPAM
+WebOb
+python-ldap
+python-memcached
+MySQL-python
 eventlet
 greenlet
 PasteDeploy
